packages/conveyer/src/Conveyer.ts

```diff
@@ -425,12 +431,17 @@ class Conveyer extends Component<ConveyerEvents> {
     });
 
     this._axes = axes;
-    if (this._options.useDrag) {
-      axes.connect(this._options.horizontal ? ["scroll", ""] : ["", "scroll"], new PanInput(scrollAreaElement, {
+    if (options.useDrag) {
+      axes.connect(options.horizontal ? ["scroll", ""] : ["", "scroll"], new PanInput(scrollAreaElement, {
         inputType: ["mouse"],
         touchAction: "auto",
       }));
     }
+    if (options.useSideWheel) {
+      axes.connect(options.horizontal ? ["scroll", ""] : ["", "scroll"], new WheelInput(scrollAreaElement, {
+        scale: 30,
```

Comment:
No additional functions of axes are required.

You will have to use the original wheels. scale must be set to 1 and useAnimation and normalized functions must also be set to false.

Reply:
Since the default scale of WheelInput is 1, it seems that we can use `useNormalized` to false.